src/spec-common/dotfiles.ts

```diff
@@ -40,16 +40,33 @@ export async function installDotfiles(params: ResolverParameters, properties: Co
 			status: 'running',
 		});
 		if (installCommand) {
-			await shellServer.exec(`# Clone & install dotfiles
+			await shellServer.exec(`# Clone & install dotfiles via '${installCommand}'
 ${createFileCommand(markerFile)} || (echo dotfiles marker found && exit 1) || exit 0
 command -v git >/dev/null 2>&1 || (echo git not found && exit 1) || exit 0
 [ -e ${targetPath} ] || ${allEnv}git clone ${repository} ${targetPath} || exit $?
```

Comment:
Actually `${targetPath}` should probably also be quoted to not break on paths with spaces. We overlooked these cases previously. Same with `${repository}`, more cases further down.